data/resources/trainings_and_courses.json

```diff
@@ -66,5 +66,18 @@
       "description": "Escal Institute of Advanced Technologies provides courses, certifications and learning materials",
       "price": "Paid"
     }
+    {
+      "name": "freeCodeCamp.org",
+      "links": [
+        {
+          "website": "https://freeCodeCamp.org"
+          "video": "https://www.youtube.com/watch?v=3Kq1MIfTWCE"
+          "video": "https://www.youtube.com/watch?v=X4eRbHgRawI"
+          "video": "https://www.youtube.com/watch?v=lZAoFs75_cs"  
+        }
+      ],
+      "description": "Entirely free courses on ethical hacking, penetration testing and development.",
```

Comment:
freeCodeCamp is more about dev. There are only 3 IT security courses:

- Information Security with HelmetJS (defensive / secure dev)
- Python for Penetration Testing (offensive / dev)
- Information Security Projects (offensive)

So it's hardly a ethical hacking & pentesting learning platform. I don't know if it is even relevant to had it to the inventory.